ajax/libs/sw-toolbox/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "sw-toolbox",
+  "filename": "sw-toolbox.js",
+  "version": "3.6.1",
+  "description": "Service Worker Toolbox provides some simple helpers for use in creating your own service workers.",
+  "keywords": [
+    "service",
+    "worker",
+    "collection",
+    "toolbox"
+  ],
+  "author": "Google's Web DevRel Team",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleChrome/sw-toolbox.git"
+  },
+  "homepage": "https://googlechrome.github.io/sw-toolbox/",
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/GoogleChrome/sw-toolbox.git",
+    "fileMap": [
+      {
+        "basePath": "",
```

Comment:
I think maybe you should use the file `sw-toolbox.js` under `lib`.
ref: https://github.com/GoogleChrome/sw-toolbox/blob/master/package.json#L12
Because the code of `sw-toolbox.js` under root is commented out.
What do you think?